ntp-proto/src/peer.rs

```diff
@@ -152,9 +152,9 @@ impl Peer {
         let time = local_clock_time;
 
         Self {
-            last_poll_interval: 2,
-            next_poll_interval: 2,
-            remote_min_poll_interval: 2,
+            last_poll_interval: PollInterval::MIN,
+            next_poll_interval: PollInterval::MIN,
+            remote_min_poll_interval: PollInterval::MIN,
```

Comment:
see below for discussion on what `PollInterval::MIN` should be. I'm not sure where these defaults come from, but `MIN` is , at least at the moment, not `2`

Reply:
I needed some initial values, and these seemed appropriate. It seems the standard doesn't specify these